LayoutTests/fast/frames/consume_transient_activation.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <script src="../../resources/js-test-pre.js"></script>
+    <script src="../../resources/ui-helper.js"></script>
+    <script>
+      description("Test consuming transient activation.");
+
+      var frame;
+      var innerFrame;
+      var innerFrame2;
+
+      function dispatchUserGesture() {
+        var x = frame.offsetLeft + frame.offsetWidth / 2;
```

Comment:
We normally use 4-space indentation in WebKit:
https://webkit.org/code-style-guidelines/#indentation-4-spaces